src/sentry/static/sentry/app/views/eventsV2/table/queryField.tsx

```diff
@@ -395,7 +403,7 @@ class QueryField extends React.Component<Props> {
       <Container className={className} gridColumns={parameters.length + 1}>
         <SelectControl
           {...selectProps}
-          styles={styles}
+          styles={!inFieldLabels ? styles : undefined}
```

Comment:
unfortunately having the flex styles on with the labels will cause the value in the selector to align to the right as if it were a tag. Because inFieldLabels are only used by the metric alert builder which doesn't have tags as a selection, I chose to disable the styles when using the inFieldLabels